modules/cluster-logging-curator-delete-index.adoc

```diff
@@ -9,9 +9,14 @@
 This file can be expanded when more functions are supported. Only delete_indices currently. When more functions are added, we can use include::modules/cluster-logging-curator-actions.adoc[leveloffset=+1] for a strict delete index files topic, if needed
 ////
 
-You can configure Curator to delete Elasticsearch data that uses the data model prior to {product-title} 4.5. You can configure per-project and global settings. Global settings apply to any project not specified. Per-project settings override global settings.
+You can configure Elasticsearch Curator to delete Elasticsearch data that uses the data model prior to {product-title} version 4.5. You can configure per-project and global settings. Global settings apply to any project not specified. Per-project settings override global settings.
 // Preserve the hard-coded "4.5" in the preceding paragraph. Do not replace it with a product version attribute.
 
+[IMPORTANT]
+====
+The Elasticsearch Curator is deprecated in {product-title} 4.7 (OpenShift Logging 5.0) and will be removed in OpenShift Logging 5.1.
```

Comment:
I'm not sure on the OpenShift doc's stance on parenthesis. For instance, it might be totally fine to include versions in parenthesis as you do here. If you wanted to drop the parenthesis, maybe use commas to set it off instead?

Reply:
Yeah - I hear you. In this case, because OCP 4.7 and OpenShift Logging 5.0 are the same thing by different names, and I'm applying this PR to OCP 4.5-4.6, I'll keep the future product name in parens. When I create the parallel PR for Logging 5.0, I'll use "OpenShift Logging 5.0" only.